sktime/transformers/resizing.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import pandas as pd
+from scipy import interpolate
+
+from sktime.transformers.base import BaseTransformer
+
+
+
+
+class TSResizeTransform(BaseTransformer):
+    """Transformer that get casual dataframe of time series and resizes 
+            Series to user length via scipy interp1d between received points.
+    """
+
+    def __init__(self, length):
+        """
+        Parameters
+        ----------
+        length : integer, the length of time series to resize to.
+        """
+        assert(length>0)
+        self.length = length
+        super(TSResizeTransform).__init__()
+        
+    def __resizeCell(self, cell):
```

Comment:
Avoid using [double underscores in Python](https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-a-single-and-a-double-underscore-before-an-object-name), private methods should have a single underscore. Please use [snake_case](https://en.wikipedia.org/wiki/Snake_case) for method names.

Reply:
done